tests/data/state_estimation/current-sensor/power-from-global-current-sensor/sym_output.json

```
@@ -0,0 +1,27 @@
+{
+  "version": "1.0",
+  "type": "sym_output",
+  "is_batch": false,
+  "attributes": {},
+  "data": {
+    "node": [
+      {"id": 1, "energized": 1, "u_pu": 1.0000000000000002, "u": 10000.000000000002, "u_angle": -5.5511151231257827e-17, "p": -152081.2411685313, "q": 51572.482671218742},
+      {"id": 2, "energized": 1, "u_pu": 1.0032295891786502, "u": 10032.295891786502, "u_angle": 0.0050437703497091899, "p": 152910.90203816551, "q": -82536.926334998439}
+    ],
+    "line": [
+      {"id": 3, "energized": 1, "p_from": -152081.2411685313, "q_from": 51572.482671218742, "i_from": 9.27153796739141, "s_from": 160587.74823825801, "p_to": 152910.90203816551, "q_to": -82536.926334998439, "i_to": 9.9999999855679569, "s_to": 173764.46176060976}
+    ],
+    "source": [
+      {"id": 4, "energized": 1, "p": -152081.2411685313, "q": 51572.482671218742, "i": 9.27153796739141, "s": 160587.74823825801, "pf": -0.94702891619660845}
+    ],
+    "sym_load": [
+      {"id": 5, "energized": 1, "p": -152910.90203816551, "q": 82536.926334998439, "i": 9.9999999855679551, "s": 173764.46176060976, "pf": -0.87998950124120556}
```

please double-check before this one
```suggestion
      {"id": 5, "energized": 1, "p": -152910.90203816551, "q": 82536.926334998439, "i": 10.0, "s": 173764.46176060976, "pf": -0.87998950124120556}
```